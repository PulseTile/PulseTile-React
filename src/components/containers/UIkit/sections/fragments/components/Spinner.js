import React from 'react';
import Spinner from '../../../../../ui-elements/Spinner/Spinner';

const SpinnerBlock = () => {
  return (
    <div id="spinner" className="ui-section">
      <strong className="ui-title">Spinner</strong>
      <div className="ui-spinner-wrap">
        <Spinner />
      </div>
    </div>
  );
};

export default SpinnerBlock;
