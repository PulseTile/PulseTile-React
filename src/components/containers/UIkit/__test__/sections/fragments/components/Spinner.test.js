import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import Spinner from '../../../../sections/fragments/components/Spinner';

Enzyme.configure({ adapter: new Adapter() });

describe('Component <Spinner />', () => {
  it('should renders Spinner with props correctly', () => {
    const component = shallow(<Spinner />);
    expect(component).toMatchSnapshot();
  });
});
