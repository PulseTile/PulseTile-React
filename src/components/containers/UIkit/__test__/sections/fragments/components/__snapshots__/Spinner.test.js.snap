// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <Spinner /> should renders Spinner with props correctly 1`] = `
<div
  className="ui-section"
  id="spinner"
>
  <strong
    className="ui-title"
  >
    Spinner
  </strong>
  <div
    className="ui-spinner-wrap"
  >
    <Spinner />
  </div>
</div>
`;
